### PR TITLE
ANC Subsequent Visit - Progress report

### PR DESCRIPTION
**Pull request by anvmn**
#268

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_826

**Comment by anvmn** [comment]
Peek 2022-06-09 11-15

**Comment by anvmn** [comment]
Selection_830

**Comment by anvmn** [comment]
Selection_832

**Comment by anvmn** [comment]
Selection_868

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Backend/Measurement/Decoder.elm [hunk 1]
```diff
@@ -2784,6 +2792,31 @@ decodeReasonForNotSendingToHC =
             )
 
 
+decodeReferralFacility : Decoder ReferralFacility
+decodeReferralFacility =
+    string
+        |> andThen
+            (\facility ->
+                case facility of
+                    "healh-center" ->
```
Comment (mariano-dagostino) [inline client/src/elm/Backend/Measurement/Decoder.elm]: Typo here? @anvmn

File: client/src/elm/Backend/Measurement/Encoder.elm [hunk 2]
```diff
@@ -2228,6 +2244,23 @@ encodeReasonForNotSendingToHC event =
                 "none"
 
 
+encodeReferralFacility : ReferralFacility -> Value
+encodeReferralFacility facility =
+    string <|
+        case facility of
+            FacilityHealthCenter ->
+                "healh-center"
```
Comment (mariano-dagostino) [inline client/src/elm/Backend/Measurement/Encoder.elm]: "health-center" ?

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good. Two minor typos

File: client/src/elm/Backend/Measurement/Decoder.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Backend/Measurement/Decoder.elm]: Indeed. Fixed. Thanks @mariano-dagostino

**Pull request merged**